src/components/PostPageDetail/index.js

```diff
@@ -85,6 +87,8 @@ const PostPageDetailIdComponent = (props) => {
   const {updateFeedContext: updateTopicContext} = usePostContextHook(CONTEXT_SOURCE.TOPIC_FEEDS);
   const {width: displayWidth} = useWindowDimensions();
 
+  const {followUnfollow} = usePostHook();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new hook `usePostHook` is introduced and its `followUnfollow` function is being used. Ensure that this hook is properly tested and handles all edge cases to avoid potential issues.
```diff
+   const {followUnfollow} = usePostHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->